examples/bert/README.md

```diff
@@ -99,6 +100,14 @@ This workflow performs BERT optimization on GPU with CUDA/TensorRT. It performs
 2. TensorRT: `TensorrtExecutionProvider`
     - *PyTorch Model -> Onnx Model -> ONNX Runtime performance tuning with trt_fp16_enable*
     Config file: [bert_trt_gpu.json](bert_trt_gpu.json)
+
+### BERT optimization with TensorRT-Model-Optimizer on CPU/GPU

```

Comment:
@jambayk should add something about deployment? like below
```Users can deploy the quantized ONNX model using TensorRT 10.x but that is not supported through Olive right now, stay tuned!`

Reply:
The comment about the deployment would probably be about ORT and not Olive? Since olive only produces the model and it gets deployed using ort, tensorrt or some other engine. I will let @EmmaNingMS comment on this.

Reply:
Vote for this version if ORT support is in the plan. "Deployment support for TensorRT-Model-Optimizer quantized models is coming soon in ORT, in the meantime try TensorRT 10.x"

What's the timeline to add the support in ORT? Will Nvidia contribute to it?

Reply:
@EmmaNingMS I will ask team about the roadmap regarding these support. Thanks.